include/circt/Dialect/FIRRTL/NLATable.h

```diff
@@ -65,6 +70,14 @@ class NLATable {
   // NLA tracking.
   void renameModule(StringAttr oldModName, StringAttr newModName);
 
+  // Rename the module names and also update the inner_sym, if the updated
+  // module name is at the leaf. renameMap is a map of old inner_sym names to
```

Comment:
What if the module name is not at the leaf?

Reply:
I realized, the rename map can also contain the inner sym name for InstanceOps, for non-leaf namepath element.
Fixed the comment.